tests/torch/sparsity/movement/test_model_saving.py

```diff
@@ -109,6 +121,145 @@ def test_same_outputs_in_torch_and_exported_onnx(self, tmp_path: Path, recipe: B
         assert np.allclose(softmax(onnx_outputs, axis=-1),
                            softmax(torch_outputs, axis=-1), atol=1e-6)
 
+    @pytest.mark.skipif(parse_version(torch.__version__) < parse_version("1.12"),
+                    reason=f"torch {torch.__version__} is not compatible with installed transformers package. "
+                            f"Some tests may fail with segmentation fault")
+    @pytest.mark.parametrize('desc', [
+        Dict(
+            nncf_weight_ratio=0.43,
```

Comment:
Removing 75% channels in the MHA and FFN doesn't give the same sparsity rate for the whole model, since dimensions are tiny and embeddings are not pruned.